src/test/kotlin/no/nav/familie/ef/sak/cucumber/steps/StepDefinitions.kt

```diff
@@ -134,6 +152,56 @@ class StepDefinitions {
                                                                          behandlingIdsToAktivitetArbeid)
     }
 
+    @Når("vedtak vedtas")
+    fun `når vedtak vedtas`() {
+
+        initialiserTilkjentYtelseOgVedtakMock()
+
+        vedtak.map {
+            val saksbehandling = saksbehandlinger.find { saksbehandling -> saksbehandling.id == it.behandlingId }
+                                 ?: error("Fant ikke saksbehandling for vedtak")
+            beregnYtelseSteg.utførSteg(saksbehandling, it.tilVedtakDto())
+        }
+    }
+
+    @Så("forvent følgende andeler lagret for behandling med id: {int}")
```

Comment:
{int} <- kult!

Reply:
For verdier som skal være lik på alle rader er dette en nyttig funksjon. Da slipper vi å repetere samme verdien hver eneste rad...